components/automate-chef-io/content/docs/desktop.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "Desktop Dashboard"
+description = "Using the Desktop Dashboard"
+date = 2018-03-26T16:01:47-07:00
+draft = false
+bref = ""
+toc = true
++++
+
+The Chef Automate _Desktop_ dashboard displays status information about all desktops connected to Chef Automate.
+Desktop information populates this dashboard after a Chef Infra Client run has executed.
+
+## Setting Up the Desktop Dashboard
+
+Enable the Desktop dashboard with: `chef-automate deploy --product automate --product infra-server --product desktop`.
```

Comment:
@vjeffrey @susanev Will update and include a link to published Chef Desktop docs after ChefConf.